core/src/Xios.cpp

```diff
@@ -16,13 +16,19 @@
  *   enable = true
  *
  * The start time, timestep, and output period will also be read from the
- * following config file entries.
+ * following config file entries. (Values shown below are the defaults, while
+ * ellipses imply that no default is set.)
  *   [model]
- *   start = ...
- *   time_step = ...
+ *   start = 1970-01-01T00:00:00Z
+ *   time_step = P0-0T01:00:00
+ *   [XiosInput]
+ *   period = ...
+ *   filename = ...
+ *   field_names = ...
  *   [XiosOutput]
  *   period = ...
  *   filename = ...
+ *   field_names = ...
```

Comment:
@AdelekeBankole note that the sections introduced here should be used when addressing #830. So it's probably best to branch off the branch used in this PR for that work.